e2e-test/src/test/java/io/cdap/plugin/errorcollector/actions/ErrorCollectorActions.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.errorcollector.actions;
+
+import io.cdap.e2e.pages.locators.CdfStudioLocators;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.SeleniumHelper;
+import io.cdap.e2e.utils.WaitHelper;
+import io.cdap.plugin.errorcollector.locators.ErrorCollectorLocators;
+import org.openqa.selenium.WebElement;
+
+
+/**
+ * Error Collector related actions.
+ */
+
+public class ErrorCollectorActions {
+
+  static {
+    SeleniumHelper.getPropertiesLocators(ErrorCollectorLocators.class);
+  }
+
+  public static void dragAndDrop(String source, String sourceTitle, int index, String sink, String sinkTitle) {
```

Comment:
None of the below actions is needed. Delete file

Reply:
File deleted.